packages/strapi-plugin-content-manager/admin/src/containers/ListSettingsView/index.js

```diff
@@ -157,44 +155,49 @@ const ListSettingsView = ({ deleteLayout, slug }) => {
 
   const [, drop] = useDrop({ accept: ItemTypes.FIELD });
 
-  const renderForm = () => (
-    <>
-      <div className="col-6" style={{ marginBottom: 4 }}>
-        <FormattedMessage id={`${pluginId}.form.Input.label`}>
-          {label => (
-            <FormattedMessage id={`${pluginId}.form.Input.label.inputDescription`}>
-              {description => (
+  const renderForm = () => {
+    const type = get(attributes, [labelToEdit, 'type'], 'text');
+    const shouldDisplaySortToggle = !['media', 'relation'].includes(type);
+
+    return (
+      <>
+        <div className="col-6" style={{ marginBottom: 4 }}>
+          <FormattedMessage id={`${pluginId}.form.Input.label`}>
+            {label => (
+              <FormattedMessage id={`${pluginId}.form.Input.label.inputDescription`}>
+                {description => (
```

Comment:
Can we use 
`formatMessage({ id: getTrad('form.Input.label.inputDescription') })` 
`formatMessage({ id: getTrad('form.Input.label') })` 
instead of the double `<FormattedMessage />` ?